src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -199,6 +243,15 @@ public static CodeActionKind GetCodeActionKindFromSuggestedActionCategoryName(st
                 _ => throw ExceptionUtilities.UnexpectedValue(categoryName)
             };
 
+        private static LSP.PriorityLevel? CodeActionPriorityToPriorityLevel(CodeActionPriority priority)
+            => priority switch
+            {
+                CodeActionPriority.Low => LSP.PriorityLevel.Low,
+                CodeActionPriority.Medium => LSP.PriorityLevel.Normal,
+                CodeActionPriority.High => LSP.PriorityLevel.High,
+                _ => null // If Roslyn doesn't assign a priority, let LSP decide the priority
```

Comment:
this doesn't seem reacahble.  throw if you hit this.

Reply:
There's a [CodeActionPriority.None](http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/CodeActions/CodeActionPriority.cs,14), in which case I thought it would make the most sense to let LSP decide the priority. I can make it an explicit case for clarity if you think that would be best.

Reply:
An explicit mapping of `None` to `null`, and a `_ => throw Unreachable` makes sense to me